test/dmarket.js

```diff
@@ -0,0 +1,1144 @@
+
+/* eslint-disable */
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-undef */
+/* eslint-disable no-console */
+/* eslint-disable func-names */
+
+const { fork } = require('child_process');
```

Comment:
Isn't this copied from the market test? Should be using the new fixtures. How easy would it be to adjust?